test/podman_pause.bats

```diff
@@ -104,7 +101,7 @@ function teardown() {
     run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} ps -a --filter id=$ctr_id"
     echo "$output"
     [ "$status" -eq 0 ]
-    run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} rm $ctr_id"
+    run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} rm -f $ctr_id"
```

Comment:
Unless this is causing issues, I'd rather not remove the '-f' here.  I'd left it off initially to make test that remove would work without it.

Reply:
Hi @TomSweeneyRedHat 
Want to confirm with you about this. As the stop command will not success to stop the paused container, the container will be running after unpause. So from current code in the rm step, we must add "-f" to it.

If the expect result is the paused container can be stopped then seems we should update the status check for stop command from "[ "$status" -eq 1 ]" to " [ "$status" -eq 0 ]".

Reply:
@ypu thanks for the background information.  I'm still puzzled why this test was working prior, you're right the rm without the -f should fail here as the container should be running and it should not be removed without the -f parameter.

I'm fine with this change as is, but think I'll tweak this test slightly after this is merged.